src/bundle/Core/Command/CleanupVersionsCommand.php

```diff
@@ -44,18 +43,15 @@ class CleanupVersionsCommand extends Command implements BackwardCompatibleComman
         self::VERSION_PUBLISHED => VersionInfo::STATUS_PUBLISHED,
     ];
 
-    /** @var \Ibexa\Contracts\Core\Repository\Repository */
-    private $repository;
+    private Repository $repository;
```

Comment:
Those can also be promoted into constructor.

```suggestion
    private readonly Repository $repository;
```